static/js/publisher/release/actions/releases.test.js

```diff
@@ -12,4 +29,234 @@ describe("releases actions", () => {
+      const store = mockStore({});
+
+      const dispatch = store.dispatch;
+      const json = {
+        success: true,
+        channel_map_tree: {
+          latest: {
+            16: {
+              amd64: [
+                {
+                  channel: "edge",
+                  info: "specific",
+                  revision: 3,
+                  version: "test"
+                }
+              ]
+            }
+          }
+        }
+      };
+
+      const release = [
+        {
+          id: 3,
+          revision: 3,
+          channels: ["latest/edge"]
+        }
+      ];
+
+      const revision = {
+        architectures: ["amd64"],
+        revision: 3
+      };
+
+      const revisions = {
+        "3": revision
+      };
+
+      handleReleaseResponse(dispatch, json, release, revisions, "latest");
+
+      const actions = store.getActions();
+
+      expect(actions[0]).toEqual({
+        payload: {
+          channel: "latest/edge",
+          revision: revision
+        },
+        type: "RELEASE_REVISION_SUCCESS"
+      });
+    });
+  });
+
+  describe("releaseRevisions", () => {
+    beforeEach(() => {
+      global.fetch = jest.fn().mockResolvedValue();
+    });
+    afterEach(() => {
+      global.fetch.mockRestore();
+    });
+
+    it("should dispatch RELEASE_REVISION_SUCCESS", () => {
+      const revision = {
+        architectures: ["amd64"],
+        revision: 3
+      };
+      const release = {
+        architecture: "amd64",
+        branch: null,
+        revision: 3,
+        risk: "edge",
+        track: "latest"
+      };
+      const store = mockStore({
+        options: {
+          snapName: "test",
+          csrfToken: "test",
+          defaultTrack: "latest"
+        },
+        pendingReleases: {
+          "3": {
+            revision: revision,
+            channels: ["latest/edge"]
+          }
+        },
+        pendingCloses: ["latest/edge"],
+        revisions: {
+          "3": revision
+        }
+      });
+
+      global.fetch = jest
+        .fn()
+        .mockResolvedValueOnce({
+          json: () => ({
+            success: true,
+            channel_map_tree: {
+              latest: {
+                16: {
+                  amd64: [
+                    {
+                      channel: "edge",
+                      info: "specific",
+                      revision: 3,
+                      version: "test"
+                    }
+                  ]
+                }
+              }
+            }
+          })
+        })
+        .mockResolvedValueOnce({
+          json: () => ({
+            success: true,
+            closed_channels: ["edge"]
+          })
+        })
+        .mockResolvedValueOnce({
+          json: () => ({
+            releases: [release],
+            revisions: [revision]
+          })
+        });
+
+      return store.dispatch(releaseRevisions()).then(() => {
+        const actions = store.getActions();
+        expect(actions[0]).toEqual({
```

Comment:
I think in case when we compare the whole array `expect(actions).toEqual([ all the actions... ])` would work as well?